code/modules/power/cable.dm

```diff
@@ -82,15 +71,20 @@
 	var/turf/T = src.loc			// hide if turf is not intact
 
 	if(level==1) hide(T.intact)
-	cable_list += src
+	cable_list += src //add it to the global cable list
 
 
-/obj/structure/cable/Destroy()						// called when a cable is deleted
+/obj/structure/cable/Destroy()					// called when a cable is deleted
 	if(powernet)
-		powernet.cut_cable(src)				// update the powernets
-	cable_list -= src
-	return ..()													// then go ahead and delete the cable
+		cut_cable_from_powernet()				// update the powernets
+	cable_list -= src							//remove it from global cable list
+	..()										// then go ahead and delete the cable

```

Comment:
Destroy() procs should always return the parent value, either by `return ..()` or `. = ..()`